docs/index.md

```diff
@@ -88,11 +88,16 @@ connection "prometheus" {
   plugin = "prometheus"
   address = "http://localhost:9090"
   metrics = ["prometheus_http_requests_.*", ".*error.*"]
+  headerName = "Authorization"
+  headerValue = "Bearer 42"
 }
 ```
 
 - `address` - HTTP address of your prometheus server
 - `metrics` - List of metric expressions to be matched against while creating dynamic metric tables
+- `headerName` - Additional header name to send to the server. If this is set, `headerValue` must also be set.
```

Comment:
Is there a default header name or value sent to Prometheus with the current plugin version? Or are these sent as `null`?

What happens if you set one but not the other and make an API request? Do you get a specific error?

Reply:
There is no authentication header sent currently, and you cannot send a request with no header name and a header value, that makes no sense. That's like saying that a map has a key, but no value, or the other way around.